package.json

```diff
@@ -19,12 +19,18 @@
     "stylelint": "stylelint src/**/*.css",
     "lint": "npm run eslint",
     "postlint": "npm run stylelint",
+    "prettier": "prettier --write \"{src,test}/**/*.{html,css,js}\"",
```

Comment:
I think this can be changed to include more files in the root:

```
"prettier": "prettier --write \"./**/*.{md,json,html,css,js.yml}\"",
```
But we have to tell prettier to ignore the package* files

Reply:
Typo in what I wrote:

```
"prettier": "prettier --write \"./**/*.{md,json,html,css,js,yml}\"",
```